model/src/channel/permission_overwrite.rs

```diff
@@ -73,17 +73,17 @@ impl Serialize for PermissionOverwrite {
     fn serialize<S: Serializer>(&self, serializer: S) -> Result<S::Ok, S::Error> {
         let mut state = serializer.serialize_struct("PermissionOverwrite", 4)?;
 
-        state.serialize_field("allow_new", &self.allow)?;
-        state.serialize_field("deny_new", &self.deny)?;
+        state.serialize_field("allow", &self.allow)?;
+        state.serialize_field("deny", &self.deny)?;
 
         match &self.kind {
             PermissionOverwriteType::Member(id) => {
                 state.serialize_field("id", &id)?;
-                state.serialize_field("type", "member")?;
+                state.serialize_field("type", &1)?;
```

Comment:
Can `self as u8` be used to avoid keeping two numbers in sync since the enum is `repr(u8)`?

Reply:
Addressed